styles/index.css

```diff
@@ -204,6 +204,10 @@ h2 {
   box-shadow: var(--primary-box-shadow);
 }
 
+.install-div--mobile {
+  display: none;
+}
+
 .btn--outline:hover {
   background-color: rgba(0, 0, 0, 0.2);
```

Comment:
You could be consistent and use either variables everywhere or color values rgba/hex/name